src/server/routers/account.tsx

```diff
@@ -204,6 +201,86 @@ export const accountRouter = createTRPCRouter({
 
       await deleteUsedCode({ ctx, token: verificationToken.token });
 
+      return user;
+    }),
+
+  deleteRequest: protectedProcedure()
+    .meta({
+      openapi: {
+        method: 'POST',
+        path: '/account/delete/request',
```

Comment:
should probably be `/account/me/delete/request`

Reply:
we don't have the /me for update email "path: '/account/update-email/'," so 3 solutions :
- Remove /me for delete
- Add /me for update email
- Remove /me for all routes (/account/me -> /account)

2 and 3 will change the routes for Start UI Native

Reply:
> Remove /me for delete

OK to me